packages/core/datadog-configuration.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+def getConfigurationFilePath() {
+    if (project.hasProperty("datadog")) {
+        if (project.datadog.configurationFilePath) {
```

Comment:
I can add a comment, the idea is to enable users to specify a path if the file is not at the root of the project. 
This could be the case for monorepos for instance.

I did something similar to what we have for specifying a service name for the sourcemaps upload: https://github.com/DataDog/dd-sdk-reactnative/blob/develop/packages/core/datadog-sourcemaps.gradle#L205
Here's how we suggest to pass it [in the docs](https://docs.datadoghq.com/real_user_monitoring/error_tracking/mobile/reactnative/#on-android-using-the-datadog-sourcemapsgradle-script): 

```groovy
project.ext.datadog = [
    serviceName: "com.my.custom.service"
]
```

So in the docs here I'd suggest something similar:

```groovy
project.ext.datadog = [
    configurationFilePath: "../datadog-configuration.json"
]
```